src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const withPartner = sex === 'm'
+    ? `${partnerName} is my wife`
+    : `${partnerName} is my husband`;
```

Comment:
in the final line the difference is only two words `wife` and `husband`
therefore, to remove repetition, it is better to move the check like this:
```
const partnerSexLabel = sex === 'm' ? 'wife' : 'husband';
const partner = isMarried ? `${partnerName} is my {partnerSexLabel}` : 'I am not married';
```